frappe/public/js/frappe/ui/filters/filter.js

```diff
@@ -90,6 +91,13 @@ frappe.ui.Filter = class {
 		this.on_change(true);
 	}
 
+	apply() {
```

Comment:
So passing parent(`flist`) to child is a generally avoided pattern (We had intentionally removed flist previously :) )
Anyway applying the filter is the role of the list rather than the individual filter, so it would be a better idea to put this function in `filter_list` that takes a filter as input. We can then call it via `on_change()` itself (`on_change()` will then need another argument `filter` in addition to `update`)